test/widgets/text_test.dart

```diff
@@ -397,13 +399,9 @@ void main() {
         addTearDown(tester.platformDispatcher.clearLocaleTestValue);
         addTearDown(tester.platformDispatcher.clearLocalesTestValue);
 
-        await tester.pumpWidget(const ZulipApp());
-        await tester.pump();
-
-        final navigator = await ZulipApp.navigator;
-        navigator.push(MaterialWidgetRoute(page: Builder(builder: (context) =>
+        await tester.pumpWidget(TestZulipApp(child: Builder(builder: (context) =>
           Text('123', style: TextStyle(textBaseline: localizedTextBaseline(context))))));
-        await tester.pumpAndSettle();
+        await tester.pump();
```

Comment:
Nice that a lot of `pumpAndSettle` calls seem to become unnecessary in favor of just `pump`.